packages/server/lib/project-base.ts

```diff
@@ -847,7 +847,10 @@ export class ProjectBase<TServer extends Server> extends EE {
 
     if (scaffoldExamples) {
       debug('will scaffold integration and fixtures folder')
-      push(scaffold.integration(cfg.integrationFolder, cfg))
+      if (!process.env.LAUNCPAD) {
```

Comment:
Uh oh... `LAUNCHPAD`. This will always be true!

Reply:
This should always be tru when running the new launchpad which is what we want. There are some system-tests for the old desktop-gui that still rely on the old scaffolding and this gating was to allow those to continue to work while allowing the new functionality for launchpad. @tgriesser am I correct here?

Reply:
I am saying you made a typo - you are missing the `N` in `LAUNCPAD`